lib/pymedphys/docs/release-notes.md

```diff
@@ -1,6 +1,3 @@
-<!-- DO NOT EDIT THIS FILE! -->
```

Comment:
is this file no longer being auto-generated?

Reply:
Just swapped which one is reference. Now the root repo level file is the copy. This was done to support the caching logic of Jupyter-Book during user edits.